resources/queries/targetedms/QCGroupingPrecursorPeptides.sql

```diff
@@ -0,0 +1,34 @@
+-- This query for Precursor Peptides displays under QC Summary menu 'Include or Exclude Peptides/Molecules'
+
+SELECT
+*,
+(CASE
+    WHEN precPep.precursorIdentifier IN (SELECT precursorIdentifier FROM targetedms.ExcludedPrecursors) THEN 'Excluded'
+    ELSE 'Included'
+END) AS markedAs
+FROM
+     (SELECT
+        (pep.PeptideModifiedSequence || ',' || cast(prec.charge AS VARCHAR) || ',' || cast(round(prec.mz, 3) AS VARCHAR)) AS precursorIdentifier, -- "key field" required for 'requireSelection' to work
+        pep.PeptideModifiedSequence AS peptideSequence,
+        -- pep.PeptideGroupId.RunId.File,
+        pep.PeptideGroupId.Label,
+        prec.charge,
+        prec.mz,
+        prec.neutralMass,
+        prec.IsotopeLabelId.Name
+        FROM
+        targetedms.Peptide pep
+        LEFT JOIN
+        targetedms.Precursor prec
+        ON
+        pep.PeptideModifiedSequence = prec.PeptideId.modifiedPeptideDisplayColumn
```

Comment:
While this ON clause will go away based on my suggestion above, we should do joins on the Id columns and not the peptide sequence columns.

Reply:
That makes sense